v1/src/main/java/com/google/cloud/teleport/spanner/ddl/Index.java

```diff
@@ -15,6 +15,8 @@
  */
 package com.google.cloud.teleport.spanner.ddl;
 
+import static com.google.cloud.teleport.spanner.common.NameUtils.quoteIdentifier;
+
 import com.google.auto.value.AutoValue;
 import com.google.cloud.spanner.Dialect;
 import com.google.common.collect.ImmutableList;
```

Comment:
Please add a comment on line 34 stating that name() will be fully qualified name if the Database contains named schema and just name otherwise, with examples.

Reply:
Also, can you add similar comment to all the other ddl classes like, Table, View etc.,